ospfd/ospf_vty.c

```diff
@@ -3365,6 +3365,82 @@ DEFUN (show_ip_ospf_instance,
 	return ret;
 }
 
+static void ospf_interface_auth_show(struct vty *vty, struct ospf_interface *oi,
+				     json_object *json, bool use_json)
+{
+	int auth_type;
+	struct ospf_if_params *params;
+	const char *auth_data = NULL;
+
+	params = IF_DEF_PARAMS(oi->ifp);
+	auth_type = OSPF_IF_PARAM(oi, auth_type);
+
+	if (auth_type == OSPF_AUTH_NOTSET)
+		return;
```

Comment:
This could be move within the `switch case` below

Reply:
This case might not be required so removed .